js-green-licenses.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packageWhitelist": [
+    /** TODO WTFPL license via PouchDB **/
```

Comment:
Not sure JSON supports comments like this?

Reply:
jsgl supports a loose json format with comments.